code/game/machinery/gameboard.dm

```diff
@@ -93,8 +93,11 @@
 	if(href_list["close"])
 		close_game()
 
-/obj/machinery/gameboard/attackby(var/obj/item/G as obj, var/mob/user as mob, params)
-	if(istype(G, /obj/item/wrench))
-		default_unfasten_wrench(user, G)
-	else if(istype(G, /obj/item/crowbar))
-		default_deconstruction_crowbar(G, ignore_panel = 1)
+/obj/machinery/gameboard/attackby(obj/item/I, mob/user, params)
+	if(default_unfasten_wrench(user, I))
+		return
+
+	if(default_deconstruction_crowbar(I, ignore_panel = TRUE))
+		return
+	else
+		return ..()
```

Comment:
This is a dangerous pattern/gives off a "bad code smell", even though it can be gotten away with "safely" here - used in other situations, this could result in the parent call being missed, or being called erroneously.
What's bugging me is the `else` being tied to only one of the bits of the `if/else` chain, instead of the crowbar being an `else if` and all of them are instead bundled together
If they *were* bundled together, forgetting a `return` in the wrench block wouldn't result in parent call, and a subsequent "double-tap"